docs/cluster-setup-guide/security/rbac.rst

```diff
@@ -85,8 +85,20 @@ workspace.
    det rbac assign-role -u mle-traffic-00 -w "Traffic Lights" WorkspaceAdmin
    det rbac assign-role -u mle-stop-00 -w "Stop Signs" WorkspaceAdmin
 
-Learn more about pre-canned roles such as ``Editor``, ``WorkspaceAdmin`` :ref:`in the section below
-<rbac-precanned>`.
+We now have two independent workspaces, one for each team.
+
+.. image:: /assets/images/rbac-example-setup-dark.png
+   :class: only-dark
+   :alt: Determined AI system architecture diagram dark mode
+
+.. image:: /assets/images/rbac-example-setup-light.png
+   :class: only-light
+   :alt: Determined AI system architecture diagram light mode
+
+.. note::
+
+   You can learn more about pre-canned roles such as ``Editor``, ``WorkspaceAdmin`` :ref:`in the
```

Comment:
Not to open a can of worms, and recognizing that English is not my firs language, I wonder whether "pre" in "pre-canned" is not redundant? Does adding "pre" to "canned" alter its meaning? I don't think it does.

Reply:
true